index.ts

```diff
@@ -609,12 +609,25 @@ export default class FreeSound {
    * downloading sounds.
    *
    * This method can set both kinds of tokens
+   * ```typescript
+   * await freeSound.setToken('your-api-key', 'oauth');
+   * ```
    */
   setToken(token: string, type?: "oauth"): string {
     this.authHeader = `${type === 'oauth' ? 'Bearer' : 'Token'} ${token}`;
     return this.authHeader;
   }
 
+  /**
```

Comment:
Missing method summary

Reply:
Ah, I didn't add method summaries for `getLoginURL(): string` and `setClientSecrets(id: string, secret: string)` because I didn't see any [search results for "getLoginURL" in the Freesound API](https://freesound.org/docs/api/search.html?q=getLoginURL), or for ["setClientSecrets"](https://freesound.org/docs/api/search.html?q=setClientSecrets).

We can add our own descriptions for these methods. It's straightforward to describe what "setClientSecrets" does; for "getLoginURL", I might say something like, "Navigate to Freesound for user login."